integration/tests/cook/test_basic.py

```diff
@@ -286,6 +288,9 @@ def test_list_jobs_by_state(self):
                 # print "%s %s" % (job['uuid'], job['status'])
                 self.assertEquals(state, job['status'])
 
+        # cancel the jobs
+        util.cancel_jobs(self.cook_url, jobs)
```

Comment:
Was this not happening before? If this is needed, should we add a `try`/`finally` for this?